pkg/porter/mixins.go

```diff
@@ -62,6 +63,41 @@ func (p *Porter) ListMixins() ([]mixin.Metadata, error) {
 	return mixins, nil
 }
 
+// SearchMixins searches the internal remote mixins list according to the provided options
+func (p *Porter) SearchMixins(opts mixin.SearchOptions) error {
+	box := packr.New("get.porter.sh/porter/pkg/mixin/remote-mixins", "../mixin/remote-mixins")
+	mixinSearcher := mixin.NewSearcher(box)
+
+	remoteMixins, err := mixinSearcher.Search(opts)
+	if err != nil {
+		return err
+	}
+
+	if len(remoteMixins) == 0 && opts.Name != "" {
+		fmt.Fprintf(p.Out, "No mixins found for %s\n", opts.Name)
```

Comment:
Should we write errors to `p.Out` instead `p.Err` ? other party who is calling `search --output json` will get parse errors

Reply:
Great call, thank you.  Updated.